test/unit/full_text_search/issue_test.rb

```diff
@@ -44,6 +44,46 @@ def test_save
                    targets.collect {|target| target.attributes.except("id")})
     end
 
+    test "Test that when you update the status of an issue, journal status is also updated" do
```

Comment:
How about checking the status before and after the update?

Reply:
As for the before update, I think the other tests have checked that it is correct.
Therefore, we are testing here focusing only on the updated status.

Reply:
In my understanding, we want to test the following part right
In the current test, we assert the other values too. So it's a little bit hard for me to know where I should focus on this test.
```
Tag.issue_status(issue.status_id).id,
```

How about stressing the status a little bit like adding the review comment as below.